From: Tobias Frost <tobi@debian.org>
Date: Mon, 20 Dec 2021 02:29:22 +0800
Subject: Use packaged catch2 library

Bug-Debian: https://bugs.debian.org/979826
---
 tests/CMakeLists.txt | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

Index: superslicer/tests/CMakeLists.txt
===================================================================
--- superslicer.orig/tests/CMakeLists.txt	2022-10-20 05:38:31.957452878 +0000
+++ superslicer/tests/CMakeLists.txt	2022-10-20 05:40:14.105459555 +0000
@@ -4,15 +4,8 @@
 set(TEST_DATA_DIR ${CMAKE_CURRENT_SOURCE_DIR}/data)
 file(TO_NATIVE_PATH "${TEST_DATA_DIR}" TEST_DATA_DIR)
 
-add_library(Catch2 INTERFACE)
-list (APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules/Catch2)
-target_include_directories(Catch2 INTERFACE ${CMAKE_CURRENT_LIST_DIR})
-add_library(Catch2::Catch2 ALIAS Catch2)
-if (APPLE)
-	# OSX builds targeting OSX 10.9 do not support new std::uncought_exception()
-	# see https://github.com/catchorg/Catch2/issues/1218
-	target_compile_definitions(Catch2 INTERFACE -DCATCH_CONFIG_NO_CPP17_UNCAUGHT_EXCEPTIONS)
-endif()
+find_package(Catch2 REQUIRED)
+
 include(Catch)
 
 set(CATCH_EXTRA_ARGS "" CACHE STRING "Extra arguments for catch2 test suites.")
@@ -25,6 +18,7 @@
 add_library(test_common INTERFACE)
 target_compile_definitions(test_common INTERFACE TEST_DATA_DIR=R"\(${TEST_DATA_DIR}\)" CATCH_CONFIG_FAST_COMPILE)
 target_link_libraries(test_common INTERFACE Catch2::Catch2)
+target_include_directories(test_common INTERFACE ${CMAKE_CURRENT_LIST_DIR})
 
 add_library(test_common_data ${COMMON_DATA_UTILS})
 target_include_directories(test_common_data PUBLIC ${TEST_DATA_DIR}/include)
